app/models/validations/sales/financial_validations.rb

```diff
@@ -52,6 +52,21 @@ def validate_percentage_bought_not_greater_than_percentage_owned(record)
     end
   end
 
+  def validate_percentage_bought_at_least_threshold(record)
+    return unless record.stairbought && record.type
+
+    threshold = if [2, 16, 18, 24].include? record.type
+                  10
+                else
+                  1
+                end
```

Comment:
do types 2, 16, 18, 24 fit inside an easily expressed category?
if so I would support making a method on SalesLog so you can do:
`threshold = record.that_category_type? ? 10 : 1`

Reply:
@arfacamble Jemma has confirmed there's no obvious category:

"Yeah sorry I don't think there is one really. And I can't see anything in common with some of the other conditional validations/displaying of questions depending on shared ownership type that we've implemented elsewhere"

One option could therefore be for us to create our own category e.g. `higher_staircasing_threshold_types` or `10_percent_staircasing_threshold_types` but I think those categories are a bit inflexible / wouldn't work well if new thresholds were introduced. Lmk if you can think of something better, otherwise I'll leave it as just an array of types.

Reply:
no I hadn't looked at what the tests were. I agree there's not much point putting them in a category only named after what is happening here.